sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/FlattenTest.java

```diff
@@ -138,7 +138,7 @@ public void testFlattenPCollectionsEmpty() {
   }
 
   @Test
-  @Category(ValidatesRunner.class)
+  @Category(NeedsRunner.class)
```

Comment:
`Flatten.pCollections()` is a primitive that should be ValidatesRunner.

`Flatten.iterables()` is unrelated.

Reply:
Ack, reverted.